content/docs/welcome-guide/setup/installing-linux.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Setting up O3DE from a deb package in Linux
+description: Learn how to install and set up Open 3D Engine (O3DE) using a deb package in Linux.
+weight: 200
+draft: true
+---
+
+To get started quickly with O3DE in Linux, download and install the deb package. The following instructions guide you through the installation process. After a successful install, you'll have a stable, pre-built version of the engine and its Gems, and you'll be ready to create new or open existing projects with the **Project Manager** tool.
+
+## Prerequisites
+
+The instructions here assume that you have met all hardware and software requirements listed in [O3DE System Requirements](../requirements#linux).
```

Comment:
The top of the requirements page is where hardware requirements are listed. We'll probably tabify this in the future, too, to list platform-specific hardware requirements.
```suggestion
The following instructions assume that you have met all hardware and software requirements listed in [O3DE System Requirements](../requirements).
```

Reply:
Does a Linux user need to perform and of the configuration in the Linux-specific section of the Requirements page? Or is this only for users who are building from source? If the installer still requires all of the configuration listed in that section, then we should make this prerequisites section a bulleted list and add a link to that section, similar to how we did here: https://o3de.org/docs/welcome-guide/setup/setup-from-github/building-linux/#prerequisites

Reply:
Only the CMake part is really needed, everything else is installed by the deb package. We cannot install cmake through the deb package because the version that is available from the sources is old. 
Going through the prerequisites doesnt fail the installation, the deb package just has dependencies and it will just skip those that are already satisfied. 
I didnt want to overcomplicate stating something like "only the CMake installation is required". Once cmake sources for ubuntu have that version, we can remove the requirement to go through the requirements.